Sources/UIPreviewCatalog/UIPreviewCatalogFilename.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import SwiftUI
+
+public protocol UIPreviewCatalogFilename {
```

Comment:
How about naming it `UIPreviewCatalogFilenameGenerater`?

Reply:
sure sounds good. alternative is to make it a closure & not use a protocol at all?

Reply:
> alternative is to make it a closure & not use a protocol at all?

I thought about it, and either one looks good to me!
I don't think it is a problem to keep using protocol.